basxconnect/core/layouts/admin_layout.py

```diff
@@ -57,4 +61,34 @@ def maintainance_package_layout(request):
 
 
 def maintenance_database_optimization(request):
-    return hg.H3("Coming Soon")
+    # get the database's size (in kilobytes)
+    # TODO: Add a button that runs the command
+    current_db_size = os.stat(os.getcwd() + "/db.sqlite3").st_size / 1000
```

Comment:
This path should be taken from django settings (check https://docs.djangoproject.com/en/dev/ref/settings/#databases)